src/loops/while_loops/hard/q6/FactOrIal.java

```diff
@@ -0,0 +1,25 @@
+package loops.while_loops.hard.q6;
+
+/*
+Write a program that will accept a non-negative number from the user and print out the factorial of that number.
```

Comment:
As this is after conditionals, we can actually just make them check if the number is >= 0.

Reply:
So basically we can have 
```
if(input < 0)
  System.out.println("Invalid input!)
```